authors/apps/comments/urls.py

```diff
@@ -0,0 +1,19 @@
+from django.urls import path
+
+from .views import (
+    CommentApiView, CommentDetailApiView, CommentReplyApiView,
+    CommentReplyDetailApiView
+)
+
+app_name = "comments"
+
+urlpatterns = [
+    path('<slug>/', CommentApiView.as_view(), name='comments'),
```

Comment:
I think we've agreed on no trailing slashes @pbkabali. Kindly look into this.

Reply:
Did we? All our urls apart from articles have trailing slashes. should I change all of them?

Reply:
I did change mine, I think it would be much helpful if we all did.

Reply:
@pbkabali I think we agreed on no trailing slashes